### PR TITLE
Lobid rdf to json 42 labels2 context

### LINKED ISSUE #42: Generate context.json from labels.json
This is what @jschnasse in hbz/lobid-rdf-to-json/pull/41 rightly suggests:

> 1. I'd suggest to exclude the generated context.json from versioning, because it is only generated from labels.json. 

It would also be nice to have that context.json serialized to a file so that it can be conveniently be copied to our web server.

### PR DESCRIPTION
**Pull request by dr0i**
Will resolve #42.
Merge after #43 and #45.

### CODE REVIEW

File: src/main/java/de/hbz/lobid/helper/EtikettMaker.java [hunk 1]
```diff
@@ -166,7 +166,6 @@ private void initMaps(InputStream labelIn) {
 		try (InputStream in = labelIn) {
 			return new ObjectMapper().readValue(in, type);
 		} catch (Exception e) {
-			e.printStackTrace();
 			throw new RuntimeException("Error during initialization!", e);
 		}
 	}
```
Comment (dr0i) [inline src/main/java/de/hbz/lobid/helper/EtikettMaker.java]: @jschnasse do you like this solution better?

**Comment by dr0i** [comment]
Creating the context from labels.json reveals that labels.json has two properties the old context didn't have:

"@id":"http://purl.org/lobid/lv#rpbSubject"
"@id":"http://rdaregistry.info/Elements/u/P60327"

@acka47 Are these ok to have in the context or not, i.e. that they must be removed in labels?

**Comment by jschnasse** [comment]
+1

**Comment by acka47** [comment]
> Creating the context from labels.json reveals that labels.json has two properties the old context didn't have:
> 
> "@id":"http://purl.org/lobid/lv#rpbSubject"
> "@id":"http://rdaregistry.info/Elements/u/P60327"

Please keep them.

**Pull request merged**